acceptance/apps_test.go

```diff
@@ -571,6 +631,8 @@ configuration:
 					return err
 				}
 
+				By("ZZZ JSON: ((" + statusJSON + "))")
+
```

Comment:
I guess this is a leftover.

Just FYI I've found the `GinkgoWriter` pretty handy to log things while testing. It has some convenience methods such as `GinkgoWriter.Println` and so on.

Reply:
Yes. :(